azurerm/data_source_api_management.go

```diff
@@ -0,0 +1,403 @@
+package azurerm
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2017-03-01/apimanagement"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceApiManagementService() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceApiManagementRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"resource_group_name": resourceGroupNameForDataSourceSchema(),
+
+			"location": locationForDataSourceSchema(),
+
+			"publisher_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"publisher_email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"sku": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"capacity": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"notification_sender_email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
can we remove the creation date, I can't think when this would be likely to be used within Terraform?